internal/document/block.go

```diff
@@ -135,10 +135,23 @@ func (b *CodeBlock) Value() []byte {
 	return b.value
 }
 
+func (b *CodeBlock) SetLine(p int, v string) {
+	b.lines[p] = v
+}
+
 func (b *CodeBlock) Category() string {
 	return b.Attributes()["category"]
 }
 
+func (b *CodeBlock) PromptEnv() bool {
+	val, err := strconv.ParseBool(b.Attributes()["promptEnv"])
+	if err != nil {
+		return false
```

Comment:
```suggestion
		return true
```

In VSCode this is true by default